daemon/rpc_login_test.go

```diff
@@ -0,0 +1,164 @@
+package daemon
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	daemonevents "github.com/NordSecurity/nordvpn-linux/daemon/events"
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+	"github.com/NordSecurity/nordvpn-linux/events"
+	"github.com/NordSecurity/nordvpn-linux/events/subs"
+	"github.com/NordSecurity/nordvpn-linux/test/category"
+	"github.com/NordSecurity/nordvpn-linux/test/mock"
+	testauth "github.com/NordSecurity/nordvpn-linux/test/mock/auth"
+	testcore "github.com/NordSecurity/nordvpn-linux/test/mock/core"
+)
+
+func TestLoginEvents(t *testing.T) {
+	category.Set(t, category.Unit)
+
+	tests := []struct {
+		name            string
+		setup           func(*daemonevents.MockPublisherSubscriber[events.DataAuthorization]) *RPC
+		loginType1      pb.LoginType
+		expectedStatus1 pb.LoginStatus
+		expectedEvent1  events.DataAuthorization
+		expectedError1  bool
```

Comment:
So, by looking on `LoginOAuth2` (which never returns an error) and how the tests are constructed looks like this can be removed, along with the respective if condition to run error/noerror assertion.